File: pythonp/variable_scope/book.py
class Book:
    name = "Avinash Nishad"     # instance / static variable

    def method(self):
        localData = 1           # local variable
        return localData


b1 = Book()
print("Static variable ", Book.name)
print("Instance variable ", b1.name)
print("Local variable ", b1.method())

File: javascriptp/OOPS/js_polymorphism/book.js
var Book = function (name, cost, author) {
  this.name = name;
  this.cost = cost;
  this.author = author;
  this.bookDetails = function () {
    console.log("Generic book");
  };
};

var Knowledge = function (name, cost, author, pages) {
  Book.call(this, name, cost, author);
  this.pages = pages;
  this.bookDetails = function () {
    console.log("Knowledge book");
  };
};

var k1 = new Knowledge("Blue Ocean", 20, "Avinash Nishad", 20);
console.log("k1", k1.bookDetails());
